Makefile

```diff
@@ -33,6 +35,9 @@ docs: docfx/docfx.exe
 docs-autobuild:
 	while true; do find docs Makefile src -type f | entr -d bash -c 'scripts/time_it make docs' ; done
 
+docs-test-examples:
+	for e in docs/examples/*.per ; do Perlang.ConsoleApp/bin/Debug/net5.0/perlang $$e ; done
+
```

Comment:
This is the essence of this PR: a new `make` rule which ensures that all examples in the docs are actually valid Perlang programs, which can run without errors. Any exceptions thrown from these scripts will now cause the CI job which runs this `make` rule to fail, making us aware of the problem as quickly as possible.